workflows/iuf/operations/vcs-upload/vcs-upload-content.yaml

```diff
@@ -74,6 +74,8 @@ spec:
                   value: "{{=jsonpath(inputs.parameters.global_params, '$.stage_params.process-media.current_product.parent_directory')}}/{{=jsonpath(inputs.parameters.global_params, '$.product_manifest.current_product.manifest.content.vcs.path')}}"
                 - name: cf_import_gitea_org
                   value: cray
+                - name: cf_import_gitea_repo
+                  value: "{{=jsonpath(inputs.parameters.global_params, '$.product_manifest.current_product.manifest.vcs.repo_name'}}"
```

Comment:
Not super confident this will work if `repo_name` isn't supplied -- wondering how argo handles optional values.